articles/expressions-and-statements.html

```diff
@@ -164,7 +164,7 @@ <h3>Some Non-Simple Statement Cases</h3>
 </li>
 <li>
 	deferred function calls and goroutine creations.
-	The two will be introduced in the next next article.
+	The two will be introduced in the next article.

```

Comment:
Ah, it is really "the next next article" here. :)
Or "the next of the next article"?

Reply:
It's little weird to say "next next" or even "the next of the next article". So I added an article link as you already did in some places.